public/app/plugins/panel/alertlist/editor.html

```diff
@@ -19,6 +19,17 @@ <h5 class="section-heading">Options</h5>
     </div>
     <gf-form-switch class="gf-form" label="Alerts from this dashboard" label-class="width-18" checked="ctrl.panel.onlyAlertsOnDashboard" on-change="ctrl.updateStateFilter()"></gf-form-switch>
   </div>
+  <div class="section gf-form-group">
+    <h5 class="section-heading">Filter</h5>
+    <div class="gf-form">
+      <span class="gf-form-label width-8">Name</span>
+      <input type="text" class="gf-form-input max-width-15" ng-model="ctrl.panel.nameFilter" ng-change="ctrl.onRefresh()" />
+    </div>
+    <div class="gf-form">
+      <span class="gf-form-label width-8">Dashboard</span>
+      <input type="text" bs-typeahead="ctrl.searchDashboards" class="gf-form-input max-width-15" ng-model="ctrl.panel.dashboardFilter"  required placeholder="Dashboard Name" ng-change="ctrl.onRefresh()" />
```

Comment:
1. We need to think about that dashboards of the same name can exists in multiple folders. We probably need to include the name of the folder in the type ahead if no one have any other better idea.
2. Shouldn't we also have a folder drop down like the dash list panel so that you can show all alerts for a specific folder? And/or have a *Alerts from this folder* checkbox similar to the *Alerts from this dashboard* checkbox.
3. *required* shouldn't be needed here